lib/jxl/enc_frame.cc

```diff
@@ -965,8 +965,8 @@ class LossyFrameEncoder {
         enc_state_->progressive_splitter.GetNumPasses());
     for (size_t i = 0; i < enc_state_->progressive_splitter.GetNumPasses();
          i++) {
-      // No coefficient reordering in Falcon mode.
-      if (enc_state_->cparams.speed_tier != SpeedTier::kFalcon) {
+      // No coefficient reordering in Falcon or faster.
```

Comment:
Still wrong :)

Reply:
Lol can you suggest a fix? We're not doing coeff reordering in falcon, are we?
(if so, then is it so fast that we don't want to disable it?)

Reply:
You switched the condition to >, so no coefficient reordering in *faster than* falcon, no?

Reply:
No, that test is testing "if slower than falcon, do something". The speed_tier in the code is actual speed (faster is higher number), so it's the inverse of the --effort in cjxl.

The difference between falcon and thunder (for vardct) is that thunder (like the old falcon) does `ClusteringType::kFastest` while the new falcon uses `ClusteringType::kFast`.

Reply:
Ah, ok :D I guess we could also do coefficient reordering for falcon now, but meh